GiniVision/Classes/ReviewViewController.swift

```diff
@@ -16,13 +16,27 @@ import UIKit
 public typealias ReviewSuccessBlock = (_ imageData: Data) -> ()
 
 /**
+ Block which will be executed each time the user rotates a picture. It contains the JPEG representation of the image including meta information about the rotated image. In the case of a PDF, it should proceed to analysis screen once it has been validated.
+ 
+ - note: Component API only.
+ */
+public typealias ReviewScreenSuccessBlock = (_ document: GiniVisionDocument, _ shouldProceedToAnalysisScreen:Bool) -> ()
```

Comment:
Still not convinced about that `shouldProceedToAnalysisScreen `...

Reply:
I'm not convinced neither, but I haven't found a better solution since document validation is done also in ReviewViewController (otherwise we could go directly to analysis when the document is a PDF).